addons/fix-pasted-scripts/addon.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "Adjust script run behavior",
+  "description": "Only run scripts on direct click, on CTRL + click, or never. (Fixes the duplicating bug)",
+  "credits": [
+    {
+      "name": "lisa_wolfgang",
+      "link": "https://github.com/lisa-wolfgang"
+    },
+    {
+      "name": "GarboMuffin",
+      "link": "https://github.com/GarboMuffin"
+    },
+    {
+      "name": "apple502j",
+      "link": "https://github.com/apple502j"
+    }
+  ],
+  "dynamicEnable": true,
+  "dynamicDisable": true,
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/projects/*"]
+    }
+  ],
+  "settings": [
+    {
+      "name": "Only run scripts...",
+      "id": "runMode",
+      "type": "select",
+      "potentialValues": [
+        {
+          "id": "default",
+          "name": "On direct click"
+        },
+        {
+          "id": "ctrl",
+          "name": "On CTRL + click"
+        },
+        {
+          "id": "fullDisable",
+          "name": "Never"
```

Comment:
I'm a little curious. Why would somebody ever want to completely disable this?

Reply:
I personally wouldn't use it, but there might be people who appreciate the option, even though there's a CTRL+click option.